subsys/debug/coredump/Kconfig

```diff
@@ -28,6 +28,17 @@ config DEBUG_COREDUMP_BACKEND_FLASH_PARTITION
 	  Core dump is saved to a flash partition with DTS alias
 	  "coredump-partition".
 
+config DEBUG_COREDUMP_BACKEND_IN_MEMORY
```

Comment:
@tbursztyka  when you have a chance, please make sure to add the two new Kconfig options to release notes for 4.2
Thanks!